seleniumJSEnv/src/main/scala/org/scalajs/jsenv/selenium/SeleniumAsyncJSRunner.scala

```diff
@@ -32,8 +32,8 @@ class SeleniumAsyncJSRunner(browserProvider: SeleniumBrowser,
   }
 
   override def stop(): Unit = synchronized {
-    if (!keepAlive || ignoreKeepAlive) {
-      future.onComplete { _ =>
+    if (browser.isOpened && !keepAlive || ignoreKeepAlive) {
```

Comment:
Well actually I'm wondering whether it shouldn't be `browser.isOpened && (!keepAlive || ignoreKeepAlive)`.